packages/common-grpc/test/service.js

```diff
@@ -1111,6 +1113,34 @@ describe('GrpcService', function() {
         fakeStream.emit('metadata');
       });
 
+      it('should emit a `request` event on each request', function(done) {
+        var fakeStream = through.obj();
+
+        ProtoService.prototype.method = function() {
+          return fakeStream;
+        };
+
+        retryRequestOverride = function(reqOpts, options) {
+          // Simulate three retries.
+          setImmediate(function() {
+            options.request();
+            options.request();
+          });
+          return options.request();
+        };
+
+        var requestStream = grpcService.requestStream(PROTO_OPTS, REQ_OPTS);
+
+        var requestEmitted = 0;
+        requestStream.on('request', function() {
+          requestEmitted++;
+          if (requestEmitted === 3) {
+            done();
+          }
+        });
+
```

Comment:
Nit: Can you remove this extra new line?

Reply:
done